mod/xrd.php

```diff
@@ -31,14 +32,14 @@ function xrd_init(&$a) {
 
 	$tpl = get_markup_template('xrd_diaspora.tpl');
 	$dspr = replace_macros($tpl,array(
-		'$baseurl' => $a->get_baseurl(),
+		'$baseurl' => App::get_baseurl(),
 		'$dspr_guid' => $r[0]['guid'],
 		'$dspr_key' => base64_encode(pemtorsa($r[0]['pubkey']))
 	));
 
 	$tpl = get_markup_template('xrd_person.tpl');
 
-	$profile_url = $a->get_baseurl().'/profile/'.$r[0]['nickname'];
+	$profile_url = App::get_baseurl().'/profile/'.$r[0]['nickname'];
 
 	if ($acct)
```

Comment:
Standards: Could you please add brackets to this conditional statement? 😊
Standards: Could you please add a space after the `if`? 😊